Versions-ios.plist.in

```diff
@@ -121,6 +122,8 @@
 		<string>11.0</string>
 		<key>14.3</key>
 		<string>11.1</string>
+		<key>14.4</key>
+		<string>11.1</string>
```

Comment:
Yes, 11.1, there was no new macOS SDK on Xcode 12.4

Reply:
I have the (old, will update) change in my local xcode12.5 branch... waiting to see if we can resurrect the bots/sims before adding newer commits